src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -439,29 +463,30 @@ DOCKER_OPTS=" -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock"
 http://localhost:2375/info
 ----
 
-Having verified that your Docker daemon is listening, run the Che container with the with `DOCKER_HOST` environment variable set to the IP address of `docker0` or `eth0` network interface. If `docker0` is running on 1.1.1.1 then:
+After you have verified that your Docker daemon is listening, run the Che container with `DOCKER_HOST` environment variable set to the IP address of `docker0` or `eth0` network interface. If `docker0` is running on 1.1.1.1:
 
 ----
 docker run -ti -e DOCKER_HOST=tcp://1.1.1.1:2375 -v /var/run/docker.sock:/var/run/docker.sock -v ~/Documents/che-data1:/data eclipse/che start
 ----
 
-Alternatively, you can save this env in `che.env` and restart Che.
+Alternatively, you can save this environment variable in the `che.env` file and restart Che.
 
 [id="proxiesfirewallsports"]
-== Proxies/Firewalls/Ports
+== Proxies/firewalls/ports
```

Comment:
This could use an active title without the slashes.

Reply:
I agree, @robin-owen . Will it make more sense to have four headings, then? Something like:
> Installing Che behind a proxy
> Running Che behind a firewall in Linux or macOS
> Running Che behind a firewall in Windows
> Limiting Che ports